Tests/CommandsTests/RunToolTests.swift

```diff
@@ -58,9 +58,19 @@ final class RunToolTests: XCTestCase {
         }
     }
 
+    func testFileDeprecation() throws {
+        fixture(name: "Miscellaneous/EchoExecutable") { path in
+            XCTAssertEqual(try execute(["Sources/secho/main.swift", "1", "2"], packagePath: path),
+                //FIME: Can't test the deprecation warning because SwiftPMProduct.execute doesn't return it
```

Comment:
Hm, maybe we're missing something in our posix_spawn. Try flushing the output stream before exec call.